draft-ietf-core-coap-tcp-tls.md

```diff
@@ -687,128 +687,105 @@ option is understood but cannot be processed, the option documents the behavior.
 
 Capability and Settings messages are used for two purposes:
 
-* Capability options advertise a capability from the sender to the recipient.
+* Capability options advertise the capabilities of the sender to the recipient. 

```

Comment:
Addressed in #43 

changed to:

Each capability option advertises one capability of the sender to the recipient.